cmd/config.go

```diff
@@ -169,3 +182,48 @@ func writeAppConfig(path string, appConfig *flyctl.AppConfig) error {
 
 	return nil
 }
+
+func saveAppV2Config(ctx context.Context, apiClient *api.Client, appName, path string) error {
+	appConfig, err := getAppV2Config(ctx, apiClient, appName)
+	// FIXME: handle situation where no releases are found, make up a fly config from the app data we have + the machine config (will be fun!)
```

Comment:
doing this turned into a slightly larger task than I expected. i'm going to make that a separate pr.